### PR TITLE
Integrated using browserstack-service

### PR DESCRIPTION
**Pull request by agarneha1331**

### CODE REVIEW

File: android/examples/run-first-test/first.conf.js [hunk 1]
```diff
@@ -9,13 +9,13 @@ exports.config = {
...
-    name: 'first_test',
+    project: "BrowserStack Samples",
+    build: 'browserstack build',
+    name: 'BStack first webdriverio-appium',
     device: 'Google Pixel 3',
     os_version: "9.0",
-    app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
-    'browserstack.debug': true
+    'browserstack.debug': true,
+    'browserstack.source': 'webdriverio-appium:sample-sdk:v1.0'
```
Comment (07souravkunda) [inline android/examples/run-first-test/first.conf.js:18]: change caps to w3c

File: android/examples/run-first-test/first.conf.js [hunk 2]
```diff
@@ -25,6 +25,14 @@ exports.config = {
   waitforTimeout: 10000,
   connectionRetryTimeout: 90000,
   connectionRetryCount: 3,
+  services: [
+    [
+      'browserstack',
+      {
+        app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>'
```
Comment (07souravkunda) [inline android/examples/run-first-test/first.conf.js:32]: give the apk/ipa path

**Review by 07souravkunda — CHANGES_REQUESTED** [review]

**Comment by 07souravkunda** [comment]
remove `run-first` test folder

File: android/examples/run-first-test/first.conf.js (hunk 2 above)
Comment (agarneha1331) [reply android/examples/run-first-test/first.conf.js]: done

File: android/examples/run-first-test/first.conf.js (hunk 1 above)
Comment (agarneha1331) [reply android/examples/run-first-test/first.conf.js]: done

File: android/examples/run-local-test/local.conf.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ exports.config = {
       'browserstack',
       { 
         browserstackLocal: true, opts: { forcelocal: false },
-        app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>'
+        app: './examples/LocalSample.apk' || 'bs://<hashed app-id>'
```
Comment (07souravkunda) [inline android/examples/run-local-test/local.conf.js:35]: here `||` is redundant. We may keep `process.env.BROWSERSTACK_APP_PATH || ./examples/LocalSample.apk`

File: android/examples/run-parallel-test/parallel.conf.js [hunk 4]
```diff
@@ -50,5 +56,6 @@ exports.config = {
 
 // Code to support common capabilities
 exports.config.capabilities.forEach(function(caps){
-  for(var i in exports.config.commonCapabilities) caps[i] = caps[i] || exports.config.commonCapabilities[i];
+  for(var i in exports.config.commonCapabilities) 
+    caps[i] = { ...caps[i], ...exports.config.commonCapabilities[i]};
```
Comment (07souravkunda) [inline android/examples/run-parallel-test/parallel.conf.js:60]: merge `bstack:options`.

**Review by 07souravkunda — CHANGES_REQUESTED** [review]

File: android/examples/run-parallel-test/parallel.conf.js (hunk 4 above)
Comment (agarneha1331) [reply android/examples/run-parallel-test/parallel.conf.js:60]: Already getting merged from line 60

**Review by 07souravkunda — APPROVED** [review]

**Pull request merged**